src/utils/networkUtils.ts

```diff
@@ -1,16 +1,14 @@
 import throttle from 'lodash/throttle';
 
+export type NetworkId = 1 | 3 | 4 | 42;
```

Comment:
now its easier to match for networkId, its not just a "number", rather it must be this ^

Reply:
you could also make it an enum

Reply:
where the name points to the number

Reply:
maybe supported networks is just an enum?

Reply:
let's grab it from snxjs when its there... agree, can be an enum.